src/AlfredoCRSF.h

```diff
@@ -19,9 +19,11 @@ class AlfredoCRSF
     void write(uint8_t b);
     void write(const uint8_t *buf, size_t len);
     void queuePacket(uint8_t addr, uint8_t type, const void *payload, uint8_t len);
+    void writePacket(uint8_t addr, uint8_t type, const void *payload, uint8_t len);
 
     // Return current channel value (1-based) in us
     int getChannel(unsigned int ch) const { return _channels[ch - 1]; }
+    const crsf_channels_t *getChanneslPacked() const { return &_channelsPacked;}
```

Comment:
this looks like a typo, should it not be getChannelsPacked?

Reply:
Oops :)
Fixed. TY